axum/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-- None.
+- **added:** Add `axum::body::to_bytes` ([#2373])
```

Comment:
You're missing a link to the PR

Reply:
Oh right, I though github links it automatically

Reply:
GitHub does I think, but the changelog might be viewed elsewhere too, for example in one's editor.